opentelemetry-etw-logs/examples/basic.rs

```diff
@@ -0,0 +1,39 @@
+//! run with `$ cargo run --example basic --all-features
+
+use opentelemetry_appender_tracing::layer;
+use opentelemetry_etw_logs::{ExporterConfig, ReentrantLogProcessor};
+use opentelemetry_sdk::logs::LoggerProvider;
+use std::collections::HashMap;
+use tracing::error;
+use tracing_subscriber::prelude::*;
+
+fn init_logger() -> LoggerProvider {
+    let exporter_config = ExporterConfig {
+        default_keyword: 1,
+        keywords_map: HashMap::new(),
+    };
+    let reenterant_processor =
+        ReentrantLogProcessor::new("test", "table".into(), None, exporter_config);
```

Comment:
would be nice to use "my-provider-name" instead of "test" so it'll self obvious what this is referring to!